src/plays/expanding-cards/Expanding-Card-Images.jsx

```diff
@@ -1,24 +1,30 @@
+import titleOne from "./images/photo-1511028931355-082bb4781053.jpeg";
```

Comment:
The file sizes are huge. Please reduce them to KBs

Reply:
Done, kindly review